_episodes/01-introduction.md

```diff
@@ -184,11 +184,11 @@ of the network.
 
 ### 8. Measure Performance
 
-Once we think the network is performing well we want to measure its performance. To do this we can use some additional data that wasn't part of the training, this is known as a validation set. There are many different methods available for doing this and which one is best depends on the type of task we are attempting. These metrics are often published as an indication of how well our network performs.
+Once we think the network is performing well we want to measure its performance. To do this we use some additional data that wasn't part of the training, this is known as a test set. There are many different methods available for measuring performance and which one is best depends on the type of task we are attempting. These metrics are often published as an indication of how well our network performs.
```

Comment:
This kind of implies that the test set isn't being used during the training process. We should at at least remove the first sentence to make this clearer.

Reply:
Yeah I get what you mean. How about replacing it with "Once we have trained the network we want to measure it's performance." ? What do you think @colinsauze ?

Reply:
I agree with @colinsauze. To me, the __test set__ is essential during training. The current formulation reads as if the test set is used after the `fit` method returns.

```suggestion
Assessing the quality of predictions of our model is essential during training and after. To do this during training, we use some additional data that wasn't part of the training in order to obtain an objective assessment. This is known as a test set. 

There are many different metrics available for measuring performance. Which one is best depends on the type of task we are attempting. These metrics are often published as an indication of how well our network performs.
```

Reply:
The terminology is not very consistent in the field. Sometimes what you're referring to is the test set, sometimes it is the other way around.

However, in the [`fit` function of keras](https://keras.io/api/models/model_training_apis/#fit-method) there is a parameter called `validation_data` to monitor performance during training. I propose we stick to the keras terminology.

Reply:
Totally in favor with @dsmits.

Reply:
Are we going to add a callout box explaining how these terms are often interchanged? Is that best here in the introduction or in episode 3 when we use it properly?

Reply:
@colinsauze I have added it to episode 3 where the term is introduced properly.

Reply:
Let me know if you approve